vocabs/licenses.ttl

```diff
@@ -11,4 +11,24 @@
 <https://opensource.org/licenses/mit-license.php> a dct:LicenseDocument ;
   rdfs:label "MIT" ;
   rdfs:comment "The MIT License" ;
+.
+
+<https://creativecommons.org/licenses/by/4.0/> a dct:LicenseDocument ;
```

Comment:
Can we get the historical versions of the CC licenses too? (i.e. less than v4.0 - and greater if available)

Reply:
[this page](https://creativecommons.org/licenses/) lists the different kinds of CC licenses they offer (but doesn't list all the versions unfortunately).

Reply:
updated